### PR TITLE
Update spec.description in 1.5.0 CSV

### PR DESCRIPTION
**Pull request by warrenvw**
Change references from OCP 4.2 to OCP 4.3.

Question: Is it appropriate to include the following in the description?

```
    - Routing and network programming for Istio components
```

### CODE REVIEW

File: olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -127,15 +127,14 @@ spec:
 
     The Serverless Operator's provided APIs such as Knative Serving
     have certain requirements with regards to the size of the underlying
-    cluster and a working installation of Service Mesh. See the [installation
-    section](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html-single/serverless/index#installing-openshift-serverless)
-    of the Serverless documentation for more info.
+    cluster and a working installation of Service Mesh. See
```
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml]: Let's ditch Service Mesh. It ain't no longer needed.

File: olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -120,22 +120,20 @@ spec:
...
     - Point-in-time snapshots of deployed code and configurations
 
     ## Prerequisites
 
     The Serverless Operator's provided APIs such as Knative Serving
     have certain requirements with regards to the size of the underlying
-    cluster and a working installation of Service Mesh. See the [installation
-    section](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html-single/serverless/index#installing-openshift-serverless)
-    of the Serverless documentation for more info.
+    cluster. See [Getting started with OpenShift Serverless](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/serverless_applications/index#serverless-getting-started)
```
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml]: Why change the link from the installation section to the getting started section?

File: olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -120,22 +120,20 @@ spec:
...
+    for more info.
 
     ## Further Information
 
     For documentation on using Knative Serving, see the
-    [serving section](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html-single/serverless/index#knative-serving_serverless-architecture) of the
-    [Serverless documentation site](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html-single/serverless/index).
+    [serving section](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/serverless_applications/index#knative-serving).
```
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml]: Why change the links here and remove the Serverless documentation site link?

File: olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml (hunk 2 above)
Comment (warrenvw) [reply olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml]: Per slack discussion, some doc sections were moved around.

File: olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml (hunk 3 above)
Comment (warrenvw) [reply olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml]: Per slack discussion, some doc sections were moved around.

**Pull request merged**